test/pubtator/pubtator.js

```diff
@@ -216,4 +218,43 @@ describe('pubtator', function () {
       }); // BioC Document
     }); // forEach
   }); // map
+  
+  describe('get', function(){
+    it('Should be null when no body returned', async function(){
+      let pmid = '38496625';
```

Comment:
I don't think this test is useful as is. e.g. If you try `39131402` (PMID exists, no pubtator data) the server returns a HTTP status 400, your code will throw an error.